gitlab/v4/objects/merge_requests.py

```diff
@@ -428,6 +456,11 @@ class ProjectMergeRequestManager(CRUDMixin, RESTManager):
         "labels": types.ListAttribute,
     }
 
+    def get(
+        self, id: Union[str, int], lazy: bool = False, **kwargs: Any
+    ) -> ProjectMergeRequest:
+        return cast(ProjectMergeRequest, super().get(id=id, lazy=lazy, **kwargs))
```

Comment:
Maybe for a follow-up it'd be good to document this in the Development docs so we can point people to this in case they are adding new managers and get hit by mypy errors?